fluentd/include/Gemfile

```diff
@@ -1,28 +1,27 @@
-# The original upstream Gemfile is: https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/addons/fluentd-elasticsearch/fluentd-es-image/Gemfile
 # To update, loosely version these packages, then run 'make update-fluentd-gemfile.lock'
 
 source 'https://rubygems.org'
 
 ### Upstream packages
-gem 'activesupport', '~>6.1'
-gem 'elasticsearch', '<7.14'
```

Comment:
@danielhoherd we pinnned elasticsearch to older version because of the issue with elasticsearch policy previously thats why its pinned to old version

Reply:
The PR for pinning it says it was pinned because of a license concern. https://github.com/astronomer/ap-vendor/pull/184

Reply:
@pgvishnuram do we know exactly what that issue is? did we make an ticket describing the problem? do we have a link to a slack thread where we discussed it?

Reply:
Has this been confirmed?

Reply:
No. We have decided to build this version and test it out anyway. We want to see if this giant leap of versions breaks, and if we can find ways to fix anything that breaks.